flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/HybridShuffleConfiguration.java

```diff
@@ -74,4 +88,82 @@ public int getMaxBuffersReadAhead() {
     public Duration getBufferRequestTimeout() {
         return bufferRequestTimeout;
     }
+
+    /**
+     * When the number of buffers that have been requested exceeds this threshold, trigger the
+     * spilling operation. Used by {@link HsSelectiveSpillingStrategy}.
+     */
+    public float getSelectiveSpillThreshold() {
+        return selectiveSpillThreshold;
+    }
+
+    /** The proportion of buffers to be spilled. Used by {@link HsSelectiveSpillingStrategy}. */
+    public float getSelectiveSpillBufferRatio() {
+        return selectiveSpillBufferRatio;
+    }
+
+    /**
+     * When the number of unSpilled buffers equal to this value, trigger the spilling operation.
+     * Used by {@link HsAllSpillingStrategy}.
+     */
+    public int getNumBuffersTriggerSpilling() {
+        return numBuffersTriggerSpilling;
+    }
```

Comment:
I'd suggest to prefix these properties with "Selective|FullStrategy", in clearly indicate that the configuration is used by a specific strategy.

Reply:
renamed